bigquery/api/BigquerySample/Program.cs

```diff
@@ -62,7 +62,7 @@ private static void Main(string[] args)
                 foreach (var row in result)
                 {
                     Console.WriteLine($"{row["url"]}: {row["view_count"]} views");
-                }
+                }            

```

Comment:
Watch out for things like this (extraneous whitespace was added, which shouldn't be there & shouldn't be part of this commit)

**Pro Tip:** _Before sending a Pull Request, scroll through the diff to double check correctness_

^--- I also do this locally before every commit, quick `git diff` locally to review changes I'm about to commit